pkgs/applications/misc/lukesmithxyz-bible/kjv.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lukesmithxyz-bible-kjv";
+  version = "stable=2022-06-01";
```

https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

```suggestion
  version = "unstable-2022-06-01";
```

---

okay.